tests/components/alarm_control_panel/test_device_action.py

```diff
@@ -266,42 +310,34 @@ def entity_reg(hass):
             ]
         },
     )
-    assert await async_setup_component(hass, DOMAIN, {DOMAIN: {CONF_PLATFORM: "test"}})
-    assert (
-        hass.states.get("alarm_control_panel.alarm_no_arm_code").state == STATE_UNKNOWN
-    )
+    assert await async_setup_component(hass, DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4564454)